src/clr-angular/data/stack-view/_variables.stack-view.scss

```diff
@@ -17,3 +17,4 @@ $clr-stack-view-stack-block-expanded-bg-color: $clr-global-selection-color;
 $clr-stack-view-stack-block-expandable-hover: $gray-light;
 $clr-stack-view-stack-block-content-text-color: inherit;
 $clr-stack-view-stack-block-expanded-text-color: $clr-black;
+$styledInputs: inputs("text", "password", "number", "email", "url", "tel", "date", "time", "datetime-local");
```

Comment:
Why was this moved into StackView?

Reply:
It is used here and in the old forms, but not used in the new forms. I moved it to keep it alive when the old forms are removed.